Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/TestCertificateContainer.kt

```diff
@@ -85,7 +85,7 @@ abstract class TestCertificateContainer {
                 get() = testCertificate.testNameAndManufactor?.let { valueSet?.getDisplayText(it) ?: it }
             override val sampleCollectedAt: Instant
                 get() = testCertificate.sampleCollectedAt
-            override val testResultAt: Instant
+            override val testResultAt: Instant?
```

Comment:
is it  possible to have a test certificate with no test date ?

Reply:
It is not marked as required here https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/test-certificate/docs/spec/dgc-test-certificate-client.md#json-format-of-a-test-certificate, I guess sampleCollectedAt is the important thing

Reply:
I thought this the date that should be used in here ?https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.2?node-id=20215%3A21115

Sample collected at is also used here 
https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.2?node-id=20215%3A21115

Reply:
I cannot see a field for the date/time of the test result in there (only when the test was conducted/ sample collected)

Reply:
I don’t see in TestCertificated.testCollectedAt either , but it is not related to your PR